src/kaleidoscope/hardware/softhruf/Splitography.cpp

```diff
@@ -27,18 +27,7 @@
 #include <Kaleidoscope.h>
 #include <avr/wdt.h>
 
-namespace kaleidoscope {
-namespace hardware {
-namespace softhruf {
-
-ATMEGA_KEYBOARD_DATA(Splitography);
-constexpr int8_t Splitography::led_count;
-
-}
-}
-}
-
+ATMEGA_KEYSCANNER_IMPLEMENTATION(softhruf::Splitography);
```

Comment:
Looking at this, I think the -macro- may go too far, removing the namespace. If we wanted to add even a single custom function, we 'd have to add the namespaces right after the macro, which defines its content inside the same namespace (but not visible to the programmer)

Also, looking at that macro, there are some things, like debounce, which I suspect we may want to make adjustable, but it's not clear to me how one would do that.

Reply:
Good point. We'll need to write the full board name then, though.... or maybe not. I have a few ideas to try.

Reply:
Turns out, we can skip the argument entirely! We'll just use `HARDWARE_IMPLEMENTATION`!

So I'll ad back the surrounding namespaces, to make it easier to add custom functions/methods/whatnot, and `ATMEGA_KEYSCANNER_IMPLEMENTATION()` will just use `HARDWARE_IMPLEMENTATION` implicitly.